stage1/init/init.go

```diff
@@ -63,21 +67,34 @@ func mirrorLocalZoneInfo(root string) {
 	_, _ = io.Copy(dest, src)
 }
 
-func main() {
-	root := "."
-	debug := len(os.Args) > 1 && os.Args[1] == "debug"
+var (
+	debug   bool
+	privNet bool
+)
+
+func init() {
+	flag.BoolVar(&debug, "debug", false, "Run in debug mode")
```

Comment:
Good catch. During git surgery (splitting it out into multiple PR's) it got lost somewhere.